code-reviews.md

```diff
@@ -0,0 +1,141 @@
+# Code Review Guide
+
+> A friendly guide for reviewing code--and not each other--at CFPB.
+
+## Why reviews?
+
+Code reviews are incredibly important step to the development process. Our goals
+when reviewing code are to:
+
+- expose bugs before they make it to production
+- ensure a concistent quality in the codebase
```

Comment:
Man, I always type that one wrong without noticing.

These aren't really individual sentences, but one group, so maybe `;` would work?